package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "taqtile-bot",
+  "version": "1.0.0",
+  "description": "Este projeto contém um CRUD feito com graphql e typescript.",
+  "main": "index.js",
+  "scripts": {
+    "start": "tsnd --transpile-only --ignore-watch node_modules src/index.ts"
+  },
+  "keywords": [],
+  "author": "walefy",
+  "license": "ISC",
+  "dependencies": {
+    "apollo-server": "^3.13.0",
```

uma prática que utilizamos é de deixar versão exatas para as dependências
```suggestion
    "apollo-server": "3.13.0",
```